draft-ietf-acme-acme.md

```diff
@@ -376,18 +376,47 @@ multiple JWS serialization formats. When sending an ACME request
 with a non-empty body, an ACME client implementation MUST use the 
 HTTP Content-Type {{!RFC7231}} header to indicate which JWS serialization format 
 is used for encapsulating the ACME request payload.
-The following Content-Type values may be used for this purpose:
+
+The list below describes the currently defined Content-Type header 
+values for indicating which JWS serialization format is used,
+along with the associated requirements. 
+A "root JWS" is the JWS used to encapsulate 
+the entire ACME request payload, and a "nested JWS" 
+is a JWS contained within the ACME request payload
+(such as the "externalAccountBinding" described in 
+{{external-account-binding}} 
+or the "key-change" object described in {{account-key-roll-over}}). 
+One of the following Content-Type header values MUST be used to indicate 
+the JWS serialization format:
 
  - "application/jose":
-   - The JWS Compact Serialization {{!RFC7515}} MUST be used. 
-   - The JWS Payload MUST NOT be detached. 
-   - The JWS Unencoded Payload Option {{!RFC7797}} MUST NOT be used. 
+   - Each root JWS MUST meet the following criteria:
+     - The JWS Compact Serialization {{!RFC7515}} MUST be used. 
+     - The JWS Payload MUST NOT be detached. 
+     - The JWS Unencoded Payload Option {{!RFC7797}} MUST NOT be used.
+   - Each nested JWS MUST be a JSON string that meets the following criteria:
+     - The JWS Compact Serialization {{!RFC7515}} MUST be used. 
```

Comment:
It's an exact duplicate of the above criteria. Can they be merged under a single header?

Reply:
Should I update the PR to read:

 - "application/jose":
   - Each root JWS and each nested JWS MUST meet the following criteria:
     - The JWS Compact Serialization {{!RFC7515}} MUST be used. 
     - The JWS Payload MUST NOT be detached. 
     - The JWS Unencoded Payload Option {{!RFC7797}} MUST NOT be used.
   - Each nested JWS MUST be a JSON string.
 - "application/jose+json":
   - Each root JWS and each nested JWS MUST meet the following criteria:
     - Either the JWS Flattened JSON Serialization {{!RFC7515}} 
       or the JWS General JSON Serialization {{!RFC7515}} MUST be used. 
     - The JWS Payload MUST NOT be detached. 
     - The JWS Unencoded Payload Option {{!RFC7797}} MUST NOT be used.
     - The JWS Unprotected Header {{!RFC7515}} SHOULD NOT be used. 
   - Each nested JWS MUST be a JSON object.

Reply:
"JSON string" is not correct. Something like this? The "payload" of the outer JWS is the base64url encoding of the (compact | general | flat) encoding of the inner JWS.

And a full example might be needed that doesn't use the pseudo-code syntax `base64url({...})` but the actual thing, even if not human readable. I'll put something together in a separate PR.